WebApplication/Startup.cs

```diff
@@ -114,8 +147,12 @@ private string CreateConnectionStringFromEnvironment()
             var user = userInfo[0];
             var password = userInfo[1];
             var database = databaseUri.LocalPath.TrimStart('/');
+            // Change this if user and password are different
+            string user = Environment.GetEnvironmentVariable("DATABASE_USERNAME") ?? "postgres";
+            string password = Environment.GetEnvironmentVariable("DATABASE_PASSWORD") ?? "root";
```

Comment:
Delete this, also a mistake from conflict resolution and this is causing your build errors